packages/core/src/App/Components/Elements/VerticalTabs/vertical-tab-headers.jsx

```diff
@@ -5,9 +5,31 @@ import {
     VerticalTabHeaderTitle }  from './vertical-tab-header.jsx';
 
 class VerticalTabHeaders extends React.PureComponent {
+    constructor(props) {
```

Comment:
`this.ref` can also be defined outside class. Unless you need it to be the same scope. if not, then the constructor can be removed and be replaced by this: 
This can be refactored into 
```jsx
classVerticalTabHeaders extends React.PureComponent {
headers = [];
// ...
```
**state.top** is not required. it can be removed.

Reply:
alright, so state isn't necessary here